CHANGELOG.md

```diff
@@ -1,108 +1,91 @@
-| Bug   | Scaffold project > yarn build fails on develop.                                                                                         | [3551][]          |
-| Bug   | storybook command failing                                                                                                               | [3567][]          |
-| Bug   | Minicart missing quantity while switching views                                                                                         | [3571][]          |
-| Bug   | Order history data not being cleared from cache on logout                                                                               | [3520][]          |
-| Bug   | Pagebuilder image content-type: mobile image is visible on desktop                                                                      | [3515][]          |
-| Bug   | Pagebuilder links force browser to load full page                                                                                       | [3524][]          |
-| Bug   | Add to Cart on category page for non-default store ends up in 404 page if store view level unique catalog product suffixes are enabled. | [3553][]          |
-| Bug   | Category page fails to load intermittently when there is Apollo related console warning.                                                | [3575][]          |
-| Bug   | Dependency on Magento was sneaked in as part of the Computed Resolver work                                                              | [16][], [6][]     |
-| Bug   | \[bug]: When clicking a submenu in the mega the submenu won't close                                                                     | [3599][]          |
-| Bug   | Fix console warning for currency                                                                                                        | [3622][]          |
-| Bug   | Remove server chunk generation from webpack                                                                                             | [3608][]          |
-| Bug   | \[Issue] Improve Gallery/AddToCartButton.js                                                                                             | [3573][]          |
-| Bug   | \[bug]: Breadcrumbs 'Shop' link redirects to '/shopnull'                                                                                | [3603][]          |
-| Bug   | Newsletter form displays two errors                                                                                                     | [3584][]          |
-| Bug   | Home page tabbing leads to page transition                                                                                              | [3625][]          |
-| Bug   | Browser Reload on any Category load all products from default category                                                                  | [3655][], [18][]  |
+
+## Highlights  
```

Comment:
Suggest grouping this list of highlights into the high level epics they are associated with:

### Personalized Content Support

<!-- General description of what was delivered in the epic -->

* [3713][]
* [3][]
* [19][] [3670][]
* [3609][]
* [2][] [3587][]
* Docs related to this epic

### PageBuilder Content optimization

<!-- General description of what was delivered in the epic -->

* [3711][]
* [3712][]
* [16][] [17][]

### ReCaptcha support

<!-- General description of what was delivered in the epic -->

* [34][] [3691][]
* [3702][]
* [20][] [3604][]

Reply:
So helpful